src/app/hud/components/ModalWindow/ModalWindow.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+import type { Node } from 'react';
+import React from 'react';
+import {
+  mainMenu,
+  wrapper,
+  labelMenu,
+} from './modalWindow.scss';
+
+type Props = {|
+  +text: string;
+  +children?: Node,
+|}
+
+const ModalWindow = ({ text, children }: Props) => (
+  <div className={wrapper}>
+    <nav className={mainMenu}>
```

Comment:
it shouldn't be `nav` because it's not menu but generic component. and change name for all styles by removing `menu` from them